src/v2/view-builder/views/TerminalView.js

```diff
@@ -132,6 +134,10 @@ const Body = BaseForm.extend({
     } else if (this.options.appState.containsMessageWithI18nKey(IDX_RETURN_LINK_OTP_ONLY)) {
       this.add(OTPInformationTerminalView);
       hasCustomView = true;
+    } else if (this.options.appState.containsMessageStartingWithI18nKey(OV_UV_ENABLE_BIOMETRICS_FASTPASS)) {
+      const options = getBiometricsErrorOptions(messagesObjs, true);
```

Comment:
can you extract this logic out and put it into `ChallengeViewUtil` and just import the view that you want to add like `OTPInformationTerminalView`

Reply:
@pradeepdewda-okta `getBiometricsErrorOptions ` is also being used in https://github.com/okta/okta-signin-widget/pull/2372/files#diff-c1cf614d130adc2fe0fe8b6d85d6f33b9777027a281f901b701cd97e2ae3cea8R128 as form error. I don't see a way to implement it as a separate view and make it inconsistent across different usage.

Reply:
@yannongli-okta what's the issue if you create a new view here ?

Reply:
@pradeepdewda-okta so after creating this view (let's call it `OktaFastPassBiometricsErrorView`), it will be used in:
1. TerminalView for SSO extension flow, as `this.add(OktaFastPassBiometricsErrorView)` under `showMessage` method.
2. ChallengeOktaVerifyFastPassView for all other flows, as `this.add(OktaFastPassBiometricsErrorView)` under `showCustomFormErrorCallout`
Is above what you recommended? I'm not sure if this is feasible technically hence asked.

Reply:
also, since we are consuming different type of objects for SSO extension and other methods, how can we get the actual error for view template data? I saw OTPInformationTerminalView is using appState but `showCustomFormErrorCallout` is relying on the `error` object passed from above layer of code.

Reply:
did you try passing the data in your view when you add it.. 

 `this.add(new OktaFastPassBiometricsErrorView( { data })`

Reply:
@jhe-okta should help you from here..